app/ui/src/Pages/Project/pages/KG/components/ResourceLists/ResourceLists.tsx

```diff
@@ -99,6 +112,12 @@ function ResourceLists({
               onLeave={onLeave}
               onChangeFilterText={setListFilterText}
               idToFullResource={idToFullResource}
+              noItems={
+                <NoItems
+                  title="No starred items yet!"
+                  subTitle="Once you favourite an item you'll see them here."
+                />
+              }
```

Comment:
Instead of passing the component, it would be simpler to pass the title and description:

```suggestion
              noItems={{
                  title: "No starred items yet!"
                  subTitle: "Once you favourite an item you'll see them here."
              }}
```

Passing a component is over customizing it as we only have one common format to display this information.

Reply:
Changed!